apis/container/v1beta1/gkecluster_types.go

```diff
@@ -257,15 +257,16 @@ type GKEClusterParameters struct {
 	// +optional
 	NetworkPolicy *NetworkPolicy `json:"networkPolicy,omitempty"`
 
-	// PodSecurityPolicyConfig: Configuration for the PodSecurityPolicy
-	// feature.
-	// +optional
-	PodSecurityPolicyConfig *PodSecurityPolicyConfig `json:"podSecurityPolicyConfig,omitempty"`
+	// NotificationConfig: Notification configuration of the cluster.
+	NotificationConfig *NotificationConfig `json:"notificationConfig,omitempty"`
 
 	// PrivateClusterConfig: Configuration for private cluster.
 	// +optional
 	PrivateClusterConfig *PrivateClusterConfigSpec `json:"privateClusterConfig,omitempty"`
 
+	// ReleaseChannel: Release channel configuration.
+	ReleaseChannel *ReleaseChannel `json:"releaseChannel,omitempty"`
+
 	// NOTE(hasheddan): ResourceLabels can only be updated via setResourceLabels
 	// https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters/setResourceLabels
```

Comment:
L#247 is a broken link . I think they changed their API. We should change it to : https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/ClusterUpdate#IntraNodeVisibilityConfig